### PR TITLE
Bugfix function name parsing in stacktrace

### PR DESCRIPTION
**Pull request by marques-work**
@ljharb @substack:

The functionName parsing logic has a typo (since bf5a750937df4920ab9e2e965be02ac69dd43f72):

```javascript
// this will split on one or more occurrences of the letter 's'
res.functionName = callDescription.split(/s+/)[0]
```

Changed it to split on `/\s+/` instead.

Also cleaned up trailing whitespaces in the penultimate commit. That part is optional, but why not?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,11 +21,11 @@ exports = module.exports = (function () {
     var lazyLoad = function () {
         return getHarness().apply(this, arguments);
     };
-
```
Comment (ljharb) [inline index.js]: Please revert these unrelated whitespace changes; that's part of the style of this project.

Comment (marques-work) [reply index.js]: @ljharb done

Comment (marques-work) [reply index.js]: @ljharb although, I'm curious -- why would you keep trailing whitespaces, such as that in https://github.com/substack/tape/blob/7261ccc40b207040f0b2586775ca12164ae93922/lib/test.js#L147 for style?

**Comment by marques-work** [comment]
@ljharb Added tests to validate function name parsing (named and anonymous). LMK if you need anything else.

File: index.js (hunk 1 above)
Comment (ljharb) [reply index.js]: It's not my decision; it's the style of the repo owner.
